app/main.py

```diff
@@ -29,4 +29,4 @@ def extract_content(url: str) -> dict:
             image_url = style[url_start:url_end].strip('"\'')
             images.append(image_url)
     text = soup.get_text()
-    return {'images': images, 'text': text}
+    return templates.TemplateResponse("results.html", {"request": request, "images": images, "text": text})
```

Comment:
Where is this template?

Reply:
It appears that I forgot to include the new `results.html` template in the PR. I will create this template and update the PR to include it. This template will be used to display the extracted images and text on a webpage.

Let's proceed with adding the `results.html` template.